lib/requests/model/PaymentCard.js

```diff
@@ -14,7 +14,10 @@ PaymentCard.body = function (data) {
         registerCard: data["registerCard"],
         cardAlias: data["cardAlias"],
         cardToken: data["cardToken"],
-        cardUserKey: data["cardUserKey"]
+        cardUserKey: data["cardUserKey"],
```

Comment:
önemli bir ayrıntı değil ama logic olarak bir arada kalmaları için önce cardUserKey'i yazıp altına cardToken'ı koyalım bence.

Reply:
done